tests/code/test_editEvent.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from app.logic.adminNewEvent import eventEdit
+from app.models.event import Event
+from datetime import *
+
+
+@pytest.mark.integration
+def test_beforeEdit():
+    eventId = 4
+    beforeEdit = Event.get_by_id(eventId)
+
+    assert beforeEdit.eventName == "First Meetup"
+
+@pytest.mark.integration
+def test_afterEdit():
+    eventFunction: editEvent(newEventData)

```

Comment:
you're not actually using your editEvent function are you? I'm not sure what this line is doing, but you are editing the event yourself in the test, not using the function.

Reply:
not sure how this test passes, actually. seems like a syntax error